.sandstorm/usr/share/gitweb/indextext.html

```diff
@@ -4,6 +4,12 @@
 <div>
   <iframe id="offer-iframe"></iframe>
 </div>
+<p>
+  <form action="publish.cgi">
+    You can also choose to publish the master branch as
+    static files, by pushing this button: <button>Publish</button>
```

Comment:
Could we also add a way to unpublish? Like, it could unlink the post-receive-hook and clear out the `/var/www` directory. I'm worried about accidentally publishing things that were meant to be private. (I understand the "public ID" would also need to leak for this to actually be a problem, but "public ID" sounds like something that could easily leak.)